tools/README.md

```diff
@@ -1,5 +0,0 @@
-# tools
-
-Tools for working with CometBFT and associated technologies.
-Documentation for these tools can be found online in the
-[CometBFT tools documentation](https://docs.cometbft.com/main/tools/).
```

Comment:
Does removing anything here affect what is written in https://docs.cometbft.com/main/tools/? If so, we should adapt the docs. From a quick glance it looks like not.